src/carma-1-tenth/run_c1t_metric_analysis.py

```diff
@@ -0,0 +1,108 @@
+import numpy as np
+import argparse, argcomplete
+import unittest
+import os
+import sys
+from tqdm import tqdm
+from rclpy.logging import set_logger_level, get_logging_severity_from_string
+from functools import partialmethod
+
+from check_distance_to_arrival import check_distance_to_arrival
+from check_message_published import check_message_published
+from check_message_timing import check_message_timing
+from check_port_drayage_ack import check_port_drayage_ack
+from plot_crosstrack_error import plot_crosstrack_error
+from plot_vehicle_speed import plot_vehicle_speed
+
+
+class C1TMetricAnalysis(unittest.TestCase):
+
+    def setUp(self):
+        self.bag_dir = sys.argv[1]
+        tqdm.__init__ = partialmethod(tqdm.__init__, disable=True)
+        set_logger_level("rosbag2_storage", get_logging_severity_from_string("FATAL"))
+        
+
+    def test_C1T_01(self):
```

Comment:
Not sure if this is a standard process but I would put a very brief comment summary before each test to concisely indicate what is being tested

Reply:
Done!